src/app/page.tsx

```diff
@@ -86,32 +91,31 @@ const futureAnn: Announcement = {
 
 export default function Landing() {
   return (
-    <div className="w-[361px] md:w-[770px] lg:w-[597px]">
-      {/* empty state (there are no announcements) */}
-      <PastAnnouncements announcements={[]} />
-      {/* empty state (there are no past announcements) */}
-      <PastAnnouncements announcements={[currentAnn, futureAnn]} />
-      {/* filled state (exactly 1 past announcement -- will show all for tablet/desktop/mobile, view all button does not render) */}
-      <PastAnnouncements announcements={[pastAnn, currentAnn]} />
-      {/* filled state (there are more than 1 past announcement but less than 3 -- will show all for tablet/desktop and cap at 1 for mobile) */}
-      <PastAnnouncements
-        announcements={[pastAnn3, pastAnn4, currentAnn, futureAnn]}
-      />
-      {/* filled state (exactly 3 past announcements -- will show all for tablet/desktop and cap at 1 for mobile) */}
-      <PastAnnouncements
-        announcements={[pastAnn, pastAnn3, pastAnn4, currentAnn, futureAnn]}
-      />
-      {/* filled state (there are more than 3 past announcements -- will cap at 3 for tablet/desktop and 1 for mobile) */}
-      <PastAnnouncements
-        announcements={[
-          pastAnn,
-          pastAnn2,
-          pastAnn3,
-          pastAnn4,
-          currentAnn,
-          futureAnn,
-        ]}
-      />
+    <div className="flex flex-col justify-center items-center bg-other-background w-full">
+      <div>
+        <div className="flex flex-col items-center gap-8 self-stretch py-16 px-4 md:py-20 md:px-8 lg:w-[1128px] lg:px-0 lg:py-[132px] lg:h-[2048px]">
+          <LandingHeader name="Vin"></LandingHeader>
+          <div className="lg:hidden flex flex-col items-start gap-8 self-stretch">
+            <CreateAnnouncementEntry></CreateAnnouncementEntry>
+            <UpcomingAnnouncements announcements={[]}></UpcomingAnnouncements>
+            <ActiveAnnouncements announcements={[]}></ActiveAnnouncements>
+            <PastAnnouncements announcements={[]}></PastAnnouncements>
+          </div>
+          <div className="max-lg:hidden flex flex-row gap-8 w-full">
+            <div className="flex flex-col gap-8 w-[499px]">
+              <CreateAnnouncementEntry></CreateAnnouncementEntry>
+              <UpcomingAnnouncements
+                announcements={[]}
+              ></UpcomingAnnouncements>
+            </div>
+            <div className="flex flex-col gap-8 w-[597px]">
+              <ActiveAnnouncements announcements={[]}></ActiveAnnouncements>
+              <PastAnnouncements announcements={[]}></PastAnnouncements>
+            </div>
+          </div>
+        </div>
+        <Footer></Footer>
+      </div>
     </div>
```

Comment:
```suggestion
    const name = "Vin";
  const announcements = [ann0, ann1, ann2, ann3, ann4, ann5];

  return (
    <div className="flex flex-col gap-16 md:gap-20 lg:w-[1128px] lg:mx-auto">
      <div className="flex flex-col gap-8 px-4 md:px-8 lg:hidden">
        <LandingHeader name={name} />
        <CreateAnnouncementEntry />
        <UpcomingAnnouncements announcements={announcements} />
        <ActiveAnnouncements announcements={announcements} />
        <PastAnnouncements announcements={announcements} />
      </div>
      <div className="max-lg:hidden flex flex-col gap-8">
        <LandingHeader name={name} />
        <div className="flex flex-row gap-8">
          <div className="flex flex-col gap-8">
            <CreateAnnouncementEntry />
            <UpcomingAnnouncements announcements={announcements} />
          </div>
          <div className="flex flex-col gap-8">
            <ActiveAnnouncements announcements={announcements} />
            <PastAnnouncements announcements={announcements} />
          </div>
        </div>
      </div>
      <Footer />
    </div>
```

Your code can be simplified much more. You should also avoid setting fixed width as much as possible. The only width that should be fixed is the main container for desktop. Take a look at my code and notice the differences.